### PR TITLE
{math}[GCC/11.3.0] GEOS v3.10.3

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u36b.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/d967ba1073df2e0e4f60e1b6b600e77c for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/e6f604670447856177eaaa27f5c96dfa for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
zen2-ubuntu-eb - Linux Ubuntu 22.04, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.10.4
See https://gist.github.com/af91d965c2850b3593481a9737969da2 for a full test report.

File: easybuild/easyconfigs/g/GEOS/GEOS-3.9.3-GCC-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+easyblock = 'ConfigureMake'
+
+name = 'GEOS'
+version = '3.9.3'
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/g/GEOS/GEOS-3.9.3-GCC-11.3.0.eb:4]: GEOS v3.10.3 is also available: https://libgeos.org/usage/download/

Do we want to bump the version or is there a reason for v3.9.3?

Comment (branfosj) [reply easybuild/easyconfigs/g/GEOS/GEOS-3.9.3-GCC-11.3.0.eb]: d0a2215

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u36b.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/5388a0df68ae890058c7ca4d3e01ba7e for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
zen2-ubuntu-eb - Linux Ubuntu 22.04, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.10.4
See https://gist.github.com/c6f65111450d7cb03fe53c1de60a418f for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/3491e974e15e0eb456f4f97dcb8621eb for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by SebastianAchilles** [comment]
Going in, thanks @branfosj!

**Pull request merged**